packages/appium/docs/en/contributing/index.md

```diff
@@ -16,8 +17,104 @@ already.
 
 ## Ways to Contribute
 
-TODO
+The project offers a variety of ways to contribute, e.g.:
+
+- contributing code
+- improving documentation
+- creating educational content (blog posts, tutorials, videos, etc.)
+- spreading the good word about the project (e.g. via Twitter)
+- filing bugs if you discover them while using Appium
+- making feature requests if you are missing something in the project
```

Comment:
triaging bugs and helping others in [Appium forum](https://discuss.appium.io) is also highly appreciated

Reply:
added 👍